rfc/rfc-3/Transaction-implementation-on-kv.md

```diff
@@ -1,140 +0,0 @@
-<!--
```

Comment:
There is another directory named `rfc/rfc-2`, it main also need to be dropped.

Reply:
<img width="620" alt="image" src="https://github.com/user-attachments/assets/92cc9068-b1c2-44f3-9610-36340f11aa87">

Reply:
For this RFC, I would suggest marking it (deprecated) to keep the history, not removing it.

Reply:
I will leave the rfc-2 as a deprecated document, not removing it.

Reply:
Should we also need to mark this file as deprecated? cc @jerryshao

Reply:
What is the meaning, how do you mark the file as deprecated?

Reply:
I have put some information above of the document.
<img width="1043" alt="image" src="https://github.com/user-attachments/assets/b35643fc-4bba-4bf4-8691-181cf5dba443">

Reply:
I think it is enough.

Reply:
> Should we also need to mark this file as deprecated? cc @jerryshao

I mean we also need to mark files in the directory `rfc3` as deprecated. @pithecuse527

Reply:
If the rfc3 also need to be marked as deprecated, not removing it, please let me know @jerryshao

Reply:
> If the rfc3 also need to be marked as deprecated, not removing it, please let me know @jerryshao

Yeah, please mark it deprecated also.